src/Table/Table.tsx

```diff
@@ -246,6 +277,24 @@ export function Table<RowShape>({
                   bodyTrElement,
                   {
                     key: getRowKey(item),
+                    ...(onRowClick
+                      ? {
+                          onClick: () => {
+                            onRowClick(item);
+                          },
+                          className: css({
+                            cursor: "pointer",
+                            "&:hover": {
+                              background:
+                                theme === "light"
+                                  ? colors.silver.light
+                                  : theme === "dark"
+                                  ? colors.midnight.darkest
+                                  : assertUnreachable(theme),
```

Comment:
Not the fault of this branch since this is the way space-kit was set up (somewhat bolted on) to work with themes at the moment, but all of these nested ternaries really don't feel great  

Hoping we can switch to something more like emotion's [ThemeProvider](https://emotion.sh/docs/theming) for at least space-kit components, or perhaps directly import Chakra's Provider (which uses emotion under the hood) with a [custom theme](https://chakra-ui.com/docs/getting-started#add-custom-theme-optional) so we could do some piecemeal migration?

(cc @Jephuff also since we'll need to port this PR to Jenn's branch later)

Reply:
Yeah... 😬 
Seems like the theming thing is something to solve when SK and Studio are co-located